rllib/evaluation/rollout_worker.py

```diff
@@ -239,13 +241,22 @@ def __init__(self,
             seed (int): Set the seed of both np and tf to this value to
                 to ensure each remote worker has unique exploration behavior.
             _fake_sampler (bool): Use a fake (inf speed) sampler for testing.
+            extra_python_environs (dict): Extra python environments need to
+                be set.
         """
         self._original_kwargs = locals().copy()
         del self._original_kwargs["self"]
 
         global _global_worker
         _global_worker = self
 
+        # set extra environs first
+        if extra_python_environs:
+            self._environs_need_reset = {}
```

Comment:
Since Ray never re-uses workers, you don't actually have to reset them. When the worker terminates, the process will be destroyed.

Reply:
This includes the driver, if using Ray Tune. It's somewhat sketchier if you're running a trainer locally not as an actor, but we can probably ignore this case.

Reply:
thanks for the suggestion, updated.